src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -5830,6 +5830,10 @@ def create_model_version_artifact_link(
             if existing_model_version_artifact_link is not None:
                 version = existing_model_version_artifact_link.version + 1
 
+            version = 1
+            if existing_model_version_artifact_link is not None:
+                version = existing_model_version_artifact_link.version + 1
+
```

Comment:
This is duplicated code, I already have a commit to remove it on my local, but will wait for CI to complete first.